Library/Formula/libxmp-lite.rb

```diff
@@ -0,0 +1,25 @@
+class LibxmpLite < Formula
+  homepage "http://xmp.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/xmp/libxmp/4.3.5/libxmp-lite-4.3.5.tar.gz"
+  sha1 "45ae99bf939f1fe2cb8c75ec76406cbb028a1bf8"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<-'EOS'.undent
+      #include <stdio.h>
+      #include <libxmp-lite/xmp.h>
+
+      int main(int argc, char** argv){
+        printf("libxmp-lite %s/%c%u\n", XMP_VERSION, *xmp_version, xmp_vercode);
```

Comment:
It doesn't have anything to do with undent... Ruby heredocs are "double-quote" interpolated (hence escape sequences are interpreted). The single quotes disable this. See: [docs](http://ruby-doc.org//core-2.2.0/doc/syntax/literals_rdoc.html#label-Here+Documents), [rtf2latex2e.rb](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/rtf2latex2e.rb#L20).

So without the quotes, any escape sequences in heredocs (like the newline at the end of the string literal there) have to be escaped again: `\\n` instead of `\n`, and if I needed a literal `'\n'`, it would have to be `\\\\n`... :confused: